core/src/test/java/com/sequenceiq/cloudbreak/service/runtimes/SupportedRuntimesTest.java

```diff
@@ -35,16 +48,49 @@ public void testSupported() {
 
     @Test
     public void testNotSupported() {
-        Whitebox.setInternalState(underTest, "latestSupportedRuntime", "7.1.0");
+        Whitebox.setInternalState(underTest, "latestSupportedRuntime", RUNTIME_710);
         // Newer versions shall not be supported
-        Assert.assertFalse(underTest.isSupported("7.2.0"));
+        Assert.assertFalse(underTest.isSupported(RUNTIME_720));
     }
 
     @Test
     public void testInvalidVersions() {
-        Whitebox.setInternalState(underTest, "latestSupportedRuntime", "7.1.0");
+        Whitebox.setInternalState(underTest, "latestSupportedRuntime", RUNTIME_710);
         // Invalid versions shall not be supported, but at least they shall not throw exception
         Assert.assertFalse(underTest.isSupported("blah"));
         Assert.assertFalse(underTest.isSupported("8"));
     }
+
+    @Test
+    public void testSupportedByDefaultImageCatalogRuntimes() throws CloudbreakImageCatalogException {
+        Whitebox.setInternalState(underTest, "latestSupportedRuntime", "");
+        when(imageCatalogService.getRuntimeVersionsFromDefault()).thenReturn(List.of(RUNTIME_7210));
```

Comment:
I think you should add a test case where it returns multiple versions, to validate it uses the latest

Reply:
Thanks. Negative case was missing as well.